Automattic/wp-calypso
# Promotion edit: Remove currency field formatting

The formatting for the currency field isn't working correctly and is
cutting off trailing zeroes. This removes the formatting for now until
we can work through how it should operate.

To Test:
1. `npm start`
2. Visit `http://calypso.localhost:3000/store/promotion/<site url>`
3. Edit any currency field to ensure that trailing zeroes after the decimal are now supported.
